test/client.rn.before.0.63.test.ts

```diff
@@ -0,0 +1,79 @@
+import * as RN from 'react-native';
```

Comment:
Why do we need this test file with this name?

Reply:
[This change on fly](https://github.com/getsentry/sentry-react-native/pull/2614/files/7abdd9d8f28600dc38b8d4b8de7f015460a52e58#diff-d02780c2f062abcf8505db670090d74c454de33dcf58974ba92e68db603208a2L123) that worked with the previous jest configuration doesn't work with the new RN settings.